src/MoviePage/index.js

```diff
@@ -38,37 +32,20 @@ function MoviePage() {
   });
 
   useEffect(() => {
-    async function getMovies() {
-      const respnse = await fetch(
-        `https://api.themoviedb.org/3/discover/movie?api_key=${process.env.MOVIE_API_KEY}&page=${page}`
-      );
-      const result = await respnse.json();
+    async function setMoviesData() {
+      const result = await getMovies(page);
       setMovies(movies.concat(result.results));
```

Comment:
I'm not 💯 sure if this is a problem or not, I know it works but as far as I recall when you want to update the state for your component considering your previous state is better to use the arrow function to access the previous state and generate the new one, for example, this could be written as:
```js
setMovies((movies) => movies.concat[result.results]);
```
it will produce the same change in the state, but right now from the top of my head I can't remember why is better to update the state that way instead of the way you are currently doing it.

🍻

Reply:
When we use `setState` I see that we follow that pattern like [here](https://medium.com/@matthew.holman/react-hooks-usestate-41ff1bf82dd) is an article on using `prevState`. I check the react docs to see if we can follow the same pattern and found [this](https://reactjs.org/docs/hooks-faq.html#how-to-get-the-previous-props-or-state). I think we can stay like this since we don't use a `prevState`.

Reply:
leave it that way it's fine and it's working 💣 🔥

Reply:
Actually gonna change it because fix this issue:
`React Hook useEffect has a missing dependency: 'movies'. Either include it or remove the dependency array. You can also do a functional update 'setMovies(m => ...)' if you only need 'movies' in the 'setMovies' call.eslint(react-hooks/exhaustive-deps)`

Reply:
Thanks!!